helm/reana/templates/reana-server.yaml

```diff
@@ -186,7 +186,14 @@ spec:
             value: {{ .Values.notifications.email_config.smtp_server | default (printf "%s-mail" (include "reana.prefix" .)) }}
           - name: REANA_EMAIL_SMTP_PORT
             value: "{{ .Values.notifications.email_config.smtp_port | default "30025" }}"
-          {{- if not .Values.debug.enabled }}
+          - name: REANA_EMAIL_SMTP_SSL
+            value: {{ .Values.notifications.email_config.smtp_ssl | default "false" | quote }}
```

Comment:
I think the `smtp_ssl` and `smtp_starttls` configurations are missing the description in the `README.md` file